api-model-v1-41/.openapi-generator/FILES

```diff
@@ -392,3 +392,34 @@ src/main/kotlin/de/gesellix/docker/remote/api/VolumeConfig.kt
 src/main/kotlin/de/gesellix/docker/remote/api/VolumeListResponse.kt
 src/main/kotlin/de/gesellix/docker/remote/api/VolumePruneResponse.kt
 src/main/kotlin/de/gesellix/docker/remote/api/VolumeUsageData.kt
+src/main/kotlin/de/gesellix/docker/remote/api/infrastructure/ApiAbstractions.kt
+src/main/kotlin/de/gesellix/docker/remote/api/infrastructure/ApiClient.kt
+src/main/kotlin/de/gesellix/docker/remote/api/infrastructure/ApiResponse.kt
+src/main/kotlin/de/gesellix/docker/remote/api/infrastructure/BigDecimalAdapter.kt
+src/main/kotlin/de/gesellix/docker/remote/api/infrastructure/BigIntegerAdapter.kt
+src/main/kotlin/de/gesellix/docker/remote/api/infrastructure/ByteArrayAdapter.kt
+src/main/kotlin/de/gesellix/docker/remote/api/infrastructure/Errors.kt
+src/main/kotlin/de/gesellix/docker/remote/api/infrastructure/LocalDateAdapter.kt
+src/main/kotlin/de/gesellix/docker/remote/api/infrastructure/LocalDateTimeAdapter.kt
+src/main/kotlin/de/gesellix/docker/remote/api/infrastructure/OffsetDateTimeAdapter.kt
+src/main/kotlin/de/gesellix/docker/remote/api/infrastructure/RequestConfig.kt
+src/main/kotlin/de/gesellix/docker/remote/api/infrastructure/RequestMethod.kt
+src/main/kotlin/de/gesellix/docker/remote/api/infrastructure/ResponseExtensions.kt
+src/main/kotlin/de/gesellix/docker/remote/api/infrastructure/Serializer.kt
+src/main/kotlin/de/gesellix/docker/remote/api/infrastructure/URIAdapter.kt
+src/main/kotlin/de/gesellix/docker/remote/api/infrastructure/UUIDAdapter.kt
+src/main/kotlin/de/gesellix/docker/remote/client/api/ConfigApi.kt
+src/main/kotlin/de/gesellix/docker/remote/client/api/ContainerApi.kt
+src/main/kotlin/de/gesellix/docker/remote/client/api/DistributionApi.kt
+src/main/kotlin/de/gesellix/docker/remote/client/api/ExecApi.kt
+src/main/kotlin/de/gesellix/docker/remote/client/api/ImageApi.kt
+src/main/kotlin/de/gesellix/docker/remote/client/api/NetworkApi.kt
+src/main/kotlin/de/gesellix/docker/remote/client/api/NodeApi.kt
+src/main/kotlin/de/gesellix/docker/remote/client/api/PluginApi.kt
+src/main/kotlin/de/gesellix/docker/remote/client/api/SecretApi.kt
+src/main/kotlin/de/gesellix/docker/remote/client/api/ServiceApi.kt
+src/main/kotlin/de/gesellix/docker/remote/client/api/SessionApi.kt
+src/main/kotlin/de/gesellix/docker/remote/client/api/SwarmApi.kt
+src/main/kotlin/de/gesellix/docker/remote/client/api/SystemApi.kt
+src/main/kotlin/de/gesellix/docker/remote/client/api/TaskApi.kt
+src/main/kotlin/de/gesellix/docker/remote/client/api/VolumeApi.kt
```

Comment:
Can you remove the changes here in `FILES`? The updateModel task isn't advanced enough to keep it clean.

Reply:
I fixed that